docs/src/components/document-renderer.tsx

```diff
@@ -90,24 +90,45 @@ const getRenderers = (
     list: ({ type, children }) => {
       if (type === 'ordered') {
         return (
-          <ol className="text-keystatic-gray-dark list-decimal list-inside">
+          <ol className="text-keystatic-gray-dark list-decimal list-inside mt-2">
             {children.map((child, index) => (
-              <li key={index}>{child}</li>
+              <li key={index} className="mb-2 leading-tight">
+                {child}
+              </li>
             ))}
           </ol>
         );
       }
       return (
-        <ul className="text-keystatic-gray-dark list-disc ml-4">
+        <ul className="text-keystatic-gray-dark list-disc ml-4 mt-2">
           {children.map((child, index) => (
-            <li key={index}>{child}</li>
+            <li key={index} className="mb-2 leading-tight">
+              {child}
+            </li>
           ))}
         </ul>
       );
     },
     divider: () => {
       return <hr className="border-keystatic-gray my-2" />;
     },
+    layout: ({ layout, children }) => (
+      <div
+        className="grid gap-6 my-2"
+        style={{
+          gridTemplateColumns: layout.map(x => `${x}fr`).join(' '),
+        }}
+      >
+        {children.map((element, i) => (
+          <div
+            key={i}
+            className="rounded-lg bg-keystatic-gray-light p-4 text-sm"
+          >
+            {element}
+          </div>
+        ))}
+      </div>
+    ),
```

Comment:
@BarnabyBishop I forgot mobile in my tweaks 📱 :) This code seems to work on my local, but you might be able to improve:

```suggestion
    layout: ({ layout, children }) => {
      const layoutStyle = `sm:grid-cols-[${layout
        .map(layoutFraction => `${layoutFraction}fr`)
        .join('_')}]`;

      return (
        <div className={`grid gap-6 my-2 grid-cols-1 ${layoutStyle}`}>
          {children.map((element, i) => (
            <div
              key={i}
              className="rounded-lg bg-keystatic-gray-light p-4 text-sm"
            >
              {element}
            </div>
          ))}
        </div>
      );
    },
```

If we add more layout options in the config then this would have to be smarter I suppose..

Reply:
Ack mobile, I parked it as I couldn't think of a solution and never came back to it. 

Is your example working on desktop? For me the `sm`/desktop styles aren't working. 

My understanding of tailwind is any of these dynamic styles in square brackets, e.g. `grid-cols-[1fr 1fr]` need to be "hardcoded" (for lack of a better word) in the className as tailwind will find these at build time and create custom css classes for it. 

If we make it dynamic like here those styles aren't known until run-time (when the app is running) so tailwind can't create these custom classes. So I don't think this will work.

We also can't just use the `style` attribute as that doesn't support media queries to my knowledge. I'll have to think of a solution here 🤔

Reply:
Hmm maybe when I tried this code I had tried hardcoded first so it was cached or something when it worked for me 🤔  

All good, I'd also be fine just hardcoding `grid-cols-1 sm:grid-cols-2` for now, or make a custom component block instead potentially, but I wonder what the recommended solution for the document field's Layout when it comes to responsiveness would be, or are they removing that (have a subtle memory of reading something somewhere but might be wrong)

Reply:
> I'd also be fine just hardcoding grid-cols-1 sm:grid-cols-2 for now

This is a good call as in our document config we only support two column layouts anyway.

> I wonder what the recommended solution for the document field's Layout when it comes to responsiveness would be

Use emotion instead of tailwind? 😁